js/src/util.js

```diff
@@ -112,6 +111,12 @@ const Util = (($) => {
         selector = element.getAttribute('href') || ''
       }
 
+      // if it's an ID
+      if (selector.charAt(0) === '#') {
+        // we escape IDs in case of special selectors (selector = '#myId:something')
+        selector = $.escapeSelector(selector).substr(1)
```

Comment:
This will fail if the user uses jQuery < 3.

Reply:
Nice catch ! :+1:

Reply:
Just keep our function like in your first patch, add a comment that this is required for jQuery < 3 compatibility and call it conditionally maybe if it's defined.

Reply:
done :+1: